### DISCUSSION TITLE
Designing a new logo for the community

### DISCUSSION THREAD

**Discussion by jarmos-san** [developersIndia/developersIndia#147] [Ideas]
The community doesn't have a logo yet, so we invite interested contributors to share your ideas & logos! Please use the comment section below to provided feedback, your logo ideas & such. And please refer to out contributing guidelines & adhere to the community code of conduct while engaging in a civil & mature conversations below.

NOTE: We'll need the assets to be in SVG format, so if your logo is accepted you'll have to either email or DM us the assets.

cc @DevanshBajaj @uragirii (please share the logos you designed over here).

**Comment by uragirii** [comment]
As per our conversation on Discord, I'm sharing the initial draft of the logo. 

The idea behind logo is simple, have a terminal over the Indian flag. To give an idea that this an Indian developer community. If you have any feedbacks let me know and I'll make changes.
![Logo_Draft#1](https://user-images.githubusercontent.com/19703138/160240706-9d1799b0-f374-44ee-889b-a2454636324c.png)

**Reply by jarmos-san** [reply]
The logo will be used pretty everywhere ranging from favicons to the navbar to Discord & Reddit. So, obviously the sizes will vary & I believe the greyish white coloured lines doesn't have enough contrast. They'll be hard to differentiate especially for the smaller logos, so can you use a different colour for those lines? Like red or something (as it is in the Discord logo)?

**Reply by Sync271** [reply]
Logos shouldn't have too much detail, should be simple and should work at smallest of sizes like icons. Feels like not everyone is going to look at the this one at smaller sizes and go "oh that's a terminal", but the Indian part of it is pretty evident.

**Reply by jarmos-san** [reply]
Yeah you're right makes sense. But it does help to have a little contrast with the background. It's not much of an addition but having a good balance of contrasts goes a long way down the road.
